src/definitions/finance.ts

```diff
@@ -18,7 +18,7 @@ export type FinanceDefinition = LocaleEntry<{
   credit_card: { [issuer: string]: string[] };
 
   /**
-   * Currencies including their name, code and symbol (e.g. `US Dollar` / `USD` / `$`).
+   * Currencies including their name, code, symbol and ISO numericCode (e.g. `US Dollar` / `USD` / `$` / '840').
```

Comment:
This should be written in plain English and not a reference to the property name:


```suggestion
   * Currencies including their name, code, symbol and ISO numerical code (e.g. `US Dollar` / `USD` / `$` / '840').
```